encoding/scenefiltering.md

```diff
@@ -21,11 +21,13 @@ fix both of these issues by using strong filtering over the entire
 video, it would likely result in detail loss in other scenes, which you
 do not want. This is where scenefiltering comes in.
 
-As always, you start by importing the Vapoursynth module and loading
+As always, you start by importing the VapourSynth module and loading
 your video source:
 
 ```py
-from vapoursynth import core
+import vapoursynth as vs  # this can look different based on your editor
+core = vs.get_core()      # the following uses VSEdit's format
```

Comment:
Yea, these two lines are automatically added when you start a new script in VSEdit. It will likely be what most people see at the top of their script.

Reply:
As discussed in #encode-autism, this should become
```
import vapoursyth as vs
core = vs.core
```